src/endpoint/s3/s3_rest.js

```diff
@@ -116,7 +117,18 @@ async function handle_request(req, res) {
     usage_report.s3_usage_info.total_calls += 1;
     usage_report.s3_usage_info[op_name] = (usage_report.s3_usage_info[op_name] || 0) + 1;
 
-
+    // Get the bucket info to fetch logging info
+    let bucket_info = null;
+    if (req.params && req.params.bucket) {
+        try {
+            bucket_info = await req.object_sdk.internal_rpc_client.bucket.read_bucket_sdk_info({ name: req.params.bucket });
+            if (bucket_info) {
+                dbg.info("bucket_info =", bucket_info);
+            }
+        } catch (err) {
+            dbg.log1("Could not get bucket info from cache", err);
```

Comment:
in general, for most errors (unless expected ones) you should use `dbg.error` and rethrow

Reply:
Done